src/screens/WebView/index.tsx

```diff
@@ -1,38 +1,116 @@
-import React from 'react'
+import React, { useContext, useLayoutEffect, useState } from 'react'
 import { useTranslation } from 'react-i18next'
 import { useLocation } from 'react-router'
+import ElectronStore from 'electron-store'
+
+import { UpdateComponent } from 'src/components/UI'
 import WebviewControls from 'src/components/UI/WebviewControls'
+import ContextProvider from 'src/state/ContextProvider'
 import { Webview } from 'src/types'
 
+const Store = window.require('electron-store')
+const configStore: ElectronStore = new Store({
+  cwd: 'store'
+})
+const { clipboard, ipcRenderer } = window.require('electron')
 import './index.css'
 
-export default function WebView() {
+type Props = {
+  isLogin?: boolean
+}
+
+type SID = {
+  sid: string
+}
+
+const USER_AGENT =
+  'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/90.0.4430.85 Safari/537.36 Edg/90.0.818.46'
+
+export default function WebView({ isLogin }: Props) {
   const { i18n } = useTranslation()
   const { pathname } = useLocation()
+  const { t } = useTranslation()
+  const { refreshLibrary } = useContext(ContextProvider)
+  const [loading, setLoading] = useState<{
+    refresh: boolean
+    message: string
+  }>({ refresh: true, message: t('loading.website', 'Loading Website') })
+  const user = configStore.get('userInfo')
+
   let lang = i18n.language
   if (i18n.language === 'pt') {
     lang = 'pt-BR'
   }
+
+  const loginUrl =
+    'https://www.epicgames.com/id/login?redirectUrl=https%3A%2F%2Fwww.epicgames.com%2Fid%2Fapi%2Fredirect'
   const epicStore = `https://www.epicgames.com/store/${lang}/`
   const wikiURL =
     'https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/wiki'
 
   const trueAsStr = 'true' as unknown as boolean | undefined
   const webview = document.querySelector('webview') as Webview
 
+  const startUrl = isLogin ? '/login' : pathname
   const urls = {
     '/epicstore': epicStore,
-    '/wiki': wikiURL
+    '/wiki': wikiURL,
+    '/login': loginUrl
   }
 
+  useLayoutEffect(() => {
+    const webview = document.querySelector('webview') as Webview
```

Comment:
You should not be using the <webview> tag as it's deprecated, I recommend using `BrowserView` instead. Plus it's better for this case because there is only 3 webviews, one for login, one for store, and one for wiki,

Reply:
Do you know how to use it to integrate with react? because I tried and could not.
Also, webview is not deprecated, it's just not recommended since chromium is always changing it and might break. But electron is warning this for years now and it wasn't broken so far.
Browserview is a `main` process so it's more complicated to deal with. I tried to style it and it was a mess.